packages/prisma/prisma/schema.prisma

```diff
@@ -22,7 +22,7 @@ model Account {
 
 model Session {
   id           Int      @id @default(autoincrement())
```

Comment:
I think all the IDs should be strings for the sake of uniformness.

Reply:
Done